pytest/lib/mocknet.py

```diff
@@ -679,19 +757,34 @@ def get_node_addr(node, port, tmp_dir):
 def update_config_file(all_nodes, tmp_dir):
     first_node = all_nodes[0]
 
-    mocknet_config_filename = os.path.join(tmp_dir, "config.json")
+
+def get_node_keys(node, tmp_dir):
+    logger.info(f'get_node_keys from {node.instance_name}')
+    node_key_filename = os.path.join(tmp_dir,
+                                     f'node_key.{node.instance_name}.json')
+    node.machine.download('/home/ubuntu/.near/node_key.json', node_key_filename)
+    with open(node_key_filename, 'r') as f:
+        node_key_json = json.load(f)
```

Comment:
We seem to be doing this a lot. Download JSON file from the machine and then load it. Perhaps this should be a separate function? If that function would use `tempfile.NamedTeporaryFile` with `delete=True` than this one wouldn’t even need `tmp_dir` simplifying the call site.